src/include/btmem.h

```diff
@@ -6,6 +6,8 @@
  * See the file LICENSE for redistribution information.
  */
 
+#define	WT_RECNO_OOB	0		/* Illegal record number */
```

Comment:
This is used in exactly one place: there are lots of 'recno == 0' and `recno != 0` checks through the code, should they be updated to use this?